### DISCUSSION TITLE
Problems encountered during PlayCover startup

### DISCUSSION THREAD

**Discussion by gegejiawei** [PlayCover/PlayCover#48] [Q&A] [answered]
There is no home screen after the program is started, sometimes it takes a long time to wait, but the program is already running in the dock bar and the network is in good condition. Are there any developers who can briefly describe all the operations performed when the software is started, so I can have a clear direction to analysis this problem. Where can I find the program log files? thanks. :)
ps: Macbook Pro 2021 14‘ (macOS Monterey 12.4)

**Comment by ghost** [comment]
If the app stays alive in the dock but no app window is shown,  right click the dock icon and select the topmost option.

Please refer to the image below.
<img width="224" alt="image" src="https://user-images.githubusercontent.com/62909766/179791076-8b58d109-03ca-44d3-8b91-71b8dd843654.png">

**Reply by gegejiawei** [reply]
Thanks, but it doesn't work,  I have tried many times.

**Reply by ghost** [reply]
Can you do a quick record of the problem? Nobody has reported this issue before. Please also provide your macOS and PlayCover information.

**Comment by gegejiawei** [comment, marked as answer]
After updating to Version 1.1.1, there is no such problem
